tools/phenomenal/nmr/nmrml2batman.xml

```diff
@@ -6,14 +6,20 @@
        <container type="docker">container-registry.phenomenal-h2020.eu/phnmnl/nmrml2batman</container>
      </requirements>
      <command><![CDATA[   
-     export indir=`dirname $nmrMLzip`;
-     mkdir -p \$indir/tmp/;
-     unzip -q -d \$indir/tmp/ $nmrMLzip;
-     nmrML2batman.R -i \$indir/tmp/
+         galaxy_data_dir="\$(dirname "${nmrMLzip}")";
+         archive_name="\$(basename "${nmrMLzip}")";
+         unzipped_path="\${galaxy_data_dir}/\${archive_name}_unzipped";
+         mkdir -p "\${unzipped_path}" && unzip -q -d "\${unzipped_path}" "${nmrMLzip}" && nmrML2batman.R -i "\${unzipped_path}";
+         exit_code=\$?;
+         rm -rf "\${unzipped_path}";
+         exit \${exit_code}
        ]]>
      </command>
+     <stdio>
```

Comment:
I dont't tend to use this construct, but probably should use it more.

Reply:
If you don't, but behaviour is (was?) to interpret any message on `stderr` as a job failure.  Kinda sucks if your tools properly log to stderr.